Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Python image to use.
```

Comment:
Consider using a slimmer base image like `python:3.9.6-alpine` to reduce the image size.